### PR TITLE
Uncompressing filter params

### PR DESCRIPTION
**Pull request by brentmiller1973**
See PR template for this PR:  https://github.com/WPMedia/fusion-news-theme-blocks/pull/463

### CODE REVIEW

**Comment by JackHowa** [comment]
is this a draft pr or ready for review? @visionbegin

**Review by JackHowa — CHANGES_REQUESTED** [review]
can you add a test case to this? I don't see any tests changed. For example, I'd expect https://github.com/WPMedia/engine-theme-sdk/blob/stable/src/components/Image/thumbor-image-url.test.ts#L3 to have a new test case with cm=t

**Comment by JackHowa** [comment]
Also, can you update either documentation in the readme or in storybook? I noticed an in-line comment. but a customer or custom block developer won't see that unless they plumb the source code

File: src/components/Image/thumbor-image-url.ts [hunk 1]
```diff
@@ -7,9 +7,25 @@ const buildThumborURL = (
   if (typeof targetImageKeyWithFilter === 'undefined' || resizerURL.length <= 1) {
     return '';
   }
-  const [targetImageKey, imageFilter = ''] = targetImageKeyWithFilter.split('=');
 
-  if (targetImageKeyWithFilter.includes('fit-in')) {
+  /**
+   * In blocks/resizer-image-block/index.js (in fusion-news-theme-blocks),
```
Comment (JackHowa) [inline src/components/Image/thumbor-image-url.ts:12]: good job adding developer documentation here

**Comment by brentmiller1973** [comment]
@JackHowa Unit test added per you request

File: src/components/Image/thumbor-image-url.test.ts [hunk 2]
```diff
@@ -14,6 +15,9 @@ describe('thumbor image url function', () => {
   it('returns a well-formed url image if all params present as expected', () => {
     expect(buildThumborURL(targetImageKey, '100x100', imageSource, resizerURL)).toBe('www.hey.resizer.com/r4YXPy4Eh2thx80bDTxRZM9Syhw=/100x100/filters:format(jpg):quality(70)/www.hey.com/ffdfdf');
   });
+  it('returns a well-formed url image if filters have been assumed defaulted (with the cm=t param) to a format of jpg and quality of 70', () => {
+    expect(buildThumborURL(targetImageKeyCompressed, '100x100', imageSource, resizerURL)).toBe('www.hey.resizer.com/r4YXPy4Eh2thx80bDTxRZM9Syhw=/100x100/filters:format(jpg):quality(70)/www.hey.com/ffdfdf');
```
Comment (JackHowa) [inline src/components/Image/thumbor-image-url.test.ts:19]: sweet

**Review by JackHowa — APPROVED** [review]
Cool thanks. did you add this fix documentation any where publicly like storybook or readme or image resizer docs (i sent you before)? Some devs will be psyched by this for perf reasons 
https://github.com/WPMedia/engine-theme-sdk/pull/97#issuecomment-702232188

**Pull request merged**